internal/resource/http/http.go

```diff
@@ -173,22 +228,3 @@ func handleResponseError(err error, w http.ResponseWriter, h *resourceHandler) {
 	}
 	w.Write([]byte(err.Error()))
 }
-
-// Note: The HTTP endpoints do not accept UID since it is quite unlikely that the user will have access to it
-func (h *resourceHandler) handleDelete(w http.ResponseWriter, r *http.Request, ctx context.Context) {
```

Comment:
This is not deleted 😂 
I grouped it right after the `handleRead` function for readability